frontend/src/libs/components/dropdown-input/dropdown-input.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  type Control,
+  type FieldErrors,
+  type FieldPath,
+  type FieldValues,
+} from 'react-hook-form';
+import { type SingleValue } from 'react-select';
+
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
+import { useCallback, useFormController } from '~/libs/hooks/hooks.js';
+import { type SelectOption } from '~/libs/types/select-option.type.js';
+
+import { Dropdown } from '../components.js';
```

Comment:
I think there's a circular dependency here. @PavloSukhinin, can you help us?

Reply:
fixed